src/components/Votes/CastVoteInput/CastVoteInput.tsx

```diff
@@ -227,22 +236,74 @@ function VoteSubmitButton({
   supportType,
   votingPower,
   missingVote,
+  proposal,
 }: {
   supportType: SupportTextProps["supportType"] | null;
   votingPower: VotingPowerData;
   missingVote: MissingVote;
+  proposal: Proposal;
 }) {
   const { write } = useCastVoteContext();
-
   const vpToDisplay = getVpToDisplay(votingPower, missingVote);
 
+  const startDate = new Intl.DateTimeFormat("en-US", {
+    month: "short",
+    day: "numeric",
+    year: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    timeZoneName: "short",
+  }).format(new Date(proposal.startTime ?? ""));
+
+  if (!supportType) {
+    return (
+      <div className="pt-3">
+        <TooltipProvider>
+          <Tooltip>
+            <TooltipTrigger className="w-full flex items-center justify-center gap-1 text-primary font-medium cursor-help">
+              <span className="flex items-center">
+                Proposal voting power{"\u00A0"}
+                <TokenAmountDisplay amount={vpToDisplay} />
+                <Image
+                  src={icons.info}
+                  alt="Info"
+                  width={16}
+                  height={16}
+                  className="ml-1"
+                />
+              </span>
+            </TooltipTrigger>
+            <TooltipContent
+              side="top"
+              align="center"
+              className="bg-neutral p-4 rounded-lg border border-line shadow-newDefault w-[calc(100vw-32px)] sm:w-[400px]"
+            >
+              <div className="flex flex-col gap-4">
+                <div>
+                  <div className="text-lg font-medium">Proposal launched</div>
+                  <div className="text-lg font-medium">{startDate}</div>
+                </div>
```

Comment:
in figma, it looks like these should be `subtitle` which is `font-semibold text-sm text-primary`